internal/env/supported_configurations.json

```diff
@@ -624,9 +627,6 @@
     "DD_TRACE_VAULT_ANALYTICS_ENABLED": [
       "A"
     ],
-    "DD_TRACE_V1_PAYLOAD_FORMAT_ENABLED": [
```

Comment:
why did this get removed? it could be that something is wrong on the other side (because I think this file is generated?), but this shouldn't be deleted.

Reply:
I don't know, this is generated code. What do I do?

Reply:
I've put the fixes to existing issues on `main` on a separate branch so that they will not be part of this PR https://github.com/DataDog/dd-trace-go/pull/4121 .